pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/text/LuceneTextIndexReader.java

```diff
@@ -150,6 +155,14 @@ public MutableRoaringBitmap getDocIds(String searchQuery) {
     }
   }
 
+  private void record(InvocationRecording recording, ImmutableRoaringBitmap matches) {
```

Comment:
Consider moving this to `TextMatchFilterOperator`

Reply:
done